operator/src/main/java/org/bf2/operator/operands/KafkaCluster.java

```diff
@@ -524,10 +524,12 @@ private JvmOptions buildZooKeeperJvmOptions(ManagedKafka managedKafka) {
 
     private KafkaExporterSpec buildKafkaExporter(ManagedKafka managedKafka) {
         ConfigMap configMap = cachedConfigMap(managedKafka, kafkaExporterLoggingConfigMapName(managedKafka));
+        String strimzi = managedKafka.getSpec().getVersions().getStrimzi();
         KafkaInstanceConfiguration config = this.configs.getConfig(managedKafka);
         KafkaExporterSpecBuilder specBuilder = new KafkaExporterSpecBuilder()
                 .withTopicRegex(".*")
                 .withGroupRegex(".*")
+                .withImage(this.overrideManager.getKafkaExporterImage(strimzi).orElse(null))
```

Comment:
```suggestion
                .withImage(this.overrideManager.getKafkaExporterImage(strimzi).orElse(null))
                .withNewTemplate()
                    .withNewPod()
                        .withImagePullSecrets(imagePullSecretManager.getOperatorImagePullSecrets(managedKafka))
                    .endPod()
                .endTemplate()
```

Reply:
Does the `withImagePullSecrets` enable Kafka exporter to build with the image?

Reply:
Updated the changes!

Reply:
> Does the `withImagePullSecrets` enable Kafka exporter to build with the image?

`withImagePullSecrets` will allow Kubernetes to pull private/secured images. Previously it only worked because the Kafka image was already downloaded and available on the node.

Reply:
Okay. Makes sense!